rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,172 @@
+
+- **Science Reviewer** is the *DCP Science PM* in coordination with *HCA Science Governance*.
+
+## Detailed Design
+
+### Proposing RFCs
+
+#### **Authors**:
+
+*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+
+  - Fork the HumanCellAtlas dcp-community repository
+  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
+  *- where "my-feature" is descriptive. Don't assign an RFC number yet.*
+  
+  - Fill in the RFC with attention to detail
+
+  - Submit a pull request.
+  
+      For software RFCs:
+      - Add the "rfc-proposed" and the appropriate software project name _(such as "Data Store")_ labels. When the RFC impacts multiple DCP software projects, then the "Architecture" label **MUST** be the project name. *If uncertain about the appropriate project name, then **_Ask a PM_** on the HCA **#dcp-project-mgmt** slack channel*
+
+      For governance RFCs:
+        
+      - Add the "rfc-proposed" and the "Governance" labels. 
+
+  - Share a link to the RFC pull request for community review on the HumanCellAtlas **#dcp** slack channel
+
+#### DCP PM:
+- Assign a **Shepherd** by completing the *Shepherd* section in the RFC template and pushing the commit. The **Shepherd** guides the RFC and its **Author(s)** through the process.
+- If there is a *Scientific "guardrails"* section in the RFC, assign the **Science Reviewer** as a reviewer of the pull request and add the "science-review-required" label.
+
+#### Shepherd:
+- Include the RFC submission in *This week in DCP* status update
+
+### Reviewing RFCs
+
+#### Approvers:
+- May assign specific reviewers in the RFC pull request
+
+#### Reviewers:
+- Add feedback to the RFC pull request comment thread
+
+#### Science Reviewer:
+- When the feedback from the Science review is addressed, replace the "science-review-required" label with "science-review-completed".
+
+#### Author: 
+- Revise the RFC pull request in response to feedback and push new commits
+
+### Shepherding RFCs
+
+#### Shepherd:
+- Monitor the community review and ensure that issues are addressed by the **Author(s)** in a reasonable time period.
+
+- When all issues are addressed and any required Science reviews are complete, summarize the review discussion for the **Approvers** in a top-level summary comment in the RFC pull request. Replace "rfc-proposed" with "rfc-final-review".
+
+- For software RFCs, share a link to the RFC pull request for approval on the HumanCellAtlas **#tech-architecture** slack channel. Add the RFC as an agenda item to the next *DCP Architecture* meeting.
+
+- For governance RFCs, share a link to the RFC pull request for approval on the HumanCellAtlas **#dcp-project-mgmt** slack channel. Add the RFC as an agenda item to the next *DCP PM* meeting.
+
+- Include the RFC final review in *This week in DCP* status update
+
+### Approving RFCs
```

Comment:
I think I might have missed it but is there a timing element here?  Like review needs to wrap up in X weeks?  I've noticed in other projects I'm working on the RFCs are just building up because there's not fixed schedule to comment on them and then finalize them.

Reply:
@tburdett  has opened an issue for timeboxes for all processes which I've added to the top-level summary comment under **Open Issues**.